vllm/model_executor/model_loader/loader.py

```diff
@@ -133,7 +133,7 @@ def _get_model_initialization_kwargs(
 
     if supports_multimodal(model_class):
         if multimodal_config is None:
-            raise ValueError("Provide vision related configurations "
+            raise ValueError("Provide multi-modal related configurations "
```

Comment:
Now looking at the previous piece of code, is it ever possible that `multimodal_config is None`? If not, then this should probably be `assert multimodal_config is not None`?

Reply:
Yeah, it can't be `None` now. It's a holdover from the previous implementation of config... let me remove this.